jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py

```diff
@@ -38,6 +36,7 @@ def pachyderm_resources():
     for repo in repos:
         client.pfs.delete_repo(repo=pfs.Repo(name=repo))
 
+    await http_client.put("/explore/unmount")
```

Comment:
Have you tried putting this after the `yield client` line within the `http_client` fixture in `conftest.py`. If that works that's probably a cleaner place for this. before the `yield` is setup code and after the `yield` is cleanup code

Reply:
Oh that's a good point, will change.